packages/widget-message-meet/src/components/activity-list/index.js

```diff
@@ -1,64 +1,54 @@
 import React, {PropTypes} from 'react';
 import classNames from 'classnames';
-import moment from 'moment';
 import ActivityItem from '../activity-item';
 import DaySeparator from '../day-separator';
 import NewMessagesSeparator from '../new-messages-separator';
 import formatDate from '../../utils/date';
 
+export const ACTIVITY_ITEM_TYPE_ACTIVITY_ITEM = `ACTIVITY_ITEM_TYPE_ACTIVITY_ITEM`;
+export const ACTIVITY_ITEM_TYPE_DAY_SEPARATOR = `ACTIVITY_ITEM_TYPE_DAY_SEPARATOR`;
+export const ACTIVITY_ITEM_TYPE_NEW_MESSAGE_SEPARATOR = `ACTIVITY_ITEM_TYPE_NEW_MESSAGE_SEPARATOR`;
```

Comment:
Didn't we say we were gonna change the names of these to something slightly different?